messages/german.json

```diff
@@ -0,0 +1,1387 @@
+{
+    "_metadata": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Metadata",
+        "fallback": "en",
+        "articles": [
+            {"pattern": "[aeiou]*", "article": "einen"},
+            {"pattern": null, "article": "ein"}
+        ],
+        "plural": [
+            {"number": 1, "index": 0},
+            {"number": null, "index": 1}
+        ],
+        "list": [
+            " und ",
+            ", ",
+            ", und "
+        ]
+    },
+    "_roles": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Roles",
+        "alpha wolf": ["Leitwolf", "Leitwölfe"],
+        "amnesiac": ["Vergessener", "Vergessener"],
+        "assassin": ["Assassine", "Assassinen"],
+        "augur": ["Weisssager", "Weisssager"],
+        "blessed villager": ["Gesegneter", "Gesegnete"],
+        "bodyguard": ["Leibwächter", "Leibwächter"],
+        "clone": ["Klon", "Klone"],
+        "crazed shaman": ["Verrückter Schamane", "Verrückte Schamanen"],
+        "cult leader": ["Kultführer", "Kultführer"],
+        "cultist": ["Kultist", "Kultisten"],
+        "cursed villager": ["Verfluchter", "Verfluchte"],
+        "demoniac": ["Dämonischer", "Demonisches"],
+        "detective": ["Detektiv", "Detektive"],
+        "doctor": ["Arzt", "Ärzte"],
+        "doomsayer": ["Apokalyptiker", "Apokalyptiker"],
+        "dullahan": ["Kopfloser Ritter", "Kopflose Ritter"],
+        "fallen angel": ["Gefallener Engel", "Gefallene Engel"],
+        "fool": ["Dummkopf", "Dummköpfe"],
+        "guardian angel": ["Schutzengel", "Schutzengel"],
+        "gunner": ["Schütze", "Schützen"],
+        "hag": ["Hexe", "Hexen"],
+        "harlot": ["Dirne", "Dirnen"],
+        "hunter": ["Jäger", "Jägern"],
+        "insomniac": ["Schlafloser", "Schlaflose"],
+        "investigator": ["Ermittler", "Ermittler"],
+        "jester": ["Hofnarr", "Hofnarr"],
+        "lycan": ["Lykaner", "Lykaner"],
+        "mad scientist": ["Verrückter Wissenschaftler", "Verrückte Wissenschaftler"],
+        "matchmaker": ["Armor", "Armor"],
+        "mayor": ["Bürgermeister", "Bürgermeister"],
+        "minion": ["Untergebener", "Untergebene"],
+        "monster": ["Monster", "Monster"],
+        "mystic": ["Mystiker", "Mystiker"],
+        "oracle": ["Orakel", "Orakel"],
+        "piper": ["Rattenfänger", "Rattenfänger"],
+        "priest": ["Priester", "Priester"],
+        "prophet": ["Prophet", "Propheten"],
+        "seer": ["Seher", "Seher"],
```

Comment:
Üblicherweise "Seherin" im deutschen. Wie wollen wir damit umgehen?

Reply:
Deutsch ist nicht so meine Welt :D ich aber ich werde das letzte in eine Seherin umändern das problem ist glaube ich dann wenn ein spieler einen Männlichen namen hat und dann als Seherin deklariert wird obwohl er männlich ist...
Deswegen hab ich noch schwierigkeiten mit den Metadaten am anfang der datei.